app/uk/gov/hmrc/essttpstubs/controllers/EligibilityController.scala

```diff
@@ -45,10 +45,13 @@ class EligibilityController @Inject() (
   }
 
   def retrieveEligibilityData: Action[EligibilityRequest] = Action.async(parse.json[EligibilityRequest]) { implicit request =>
+    logger.info(s"Request body for request: ${request.uri} [ ${Json.prettyPrint(Json.toJson(request.body))} ]")
     for {
       eligibilityResponse: Option[JsObject] <- eligibilityService.eligibilityData(request.body)
     } yield eligibilityResponse match {
-      case Some(validResponse) => Ok(Json.toJson(validResponse))
+      case Some(validResponse) =>
+        logger.info(s"Response body for request to ${request.uri}: [ ${Json.prettyPrint(Json.toJson(validResponse))} ]")
```

Comment:
we may as well log the not found response just below as well

Reply:
oh true